doc/source/ray-air/benchmarks.rst

```diff
@@ -126,8 +126,87 @@ XGBoost parameters were kept as defaults for xgboost==1.6.1 this task.
       - `python xgboost_benchmark.py --size 100GB`
 
 
+GPU image batch prediction
+----------------------------------------------------
+
+This task uses the BatchPredictor module to process different amounts of data
+using a Pytorch pre-trained ResNet model.
+
+We test out the performance across different cluster sizes and data sizes.
+
+- `GPU image batch prediction script`_
+
+.. list-table::
+
+    * - **Cluster Setup**
```

Comment:
Let's condense the table into 4 rows (as in this PR: https://github.com/ray-project/ray/pull/26633)
- Cluster Setup: 1 g3.8xlarge node
- Data Size: 1 GB (1623 images)
- Performance: 72.59 s (22.3 images/s)
- Command: ...

Reply:
fixed!